src/main/java/uk/gov/hmcts/dts/fact/services/admin/AdminCourtAddressService.java

```diff
@@ -376,4 +378,15 @@ private void deleteExistingSecondaryAddressTypes(String slug) {
                                                                        .collect(toList()));
     }
 
+    /**
+    * Validate court address epim ids.
+    * @param courtAddresses The court addresses
+    * @return epim id if invalid else null
+     */
+    public String validateCourtAddressEpimId(List<CourtAddress> courtAddresses) {
+        if (!CollectionUtils.isEmpty(courtAddresses)) {
+            return validationService.validateEpimIds(courtAddresses);
+        }
+        return null;
```

Comment:
Please don't return nulls. and you could probably combine this and validateEpimIds into one method. 

I think you could literally check if its invalid based on the criteria and if so, then raise an exception at that point. Can be confusing otherwise to check if something is or isn't null and then act based on that

Reply:
changed